server/subscriptions.go

```diff
@@ -102,6 +98,41 @@ func (p *Plugin) Subscribe(ctx context.Context, githubClient *github.Client, use
 	return nil
 }
 
+func (p *Plugin) SubscribeOrg(ctx context.Context, githubClient *github.Client, userId, org, channelID, features string) error {
+	if org == "" {
+		return fmt.Errorf("Invalid organization")
+	}
+	if err := p.checkOrg(org); err != nil {
+		return err
+	}
+
+	listOrgOptions := github.RepositoryListByOrgOptions{
+		Type: "all",
+	}
+	repos, _, err := githubClient.Repositories.ListByOrg(ctx, org, &listOrgOptions)
```

Comment:
The github provides Pages results by default. Hence this will only fetch the first 100 (I think) repositories. See https://github.com/google/go-github#pagination for more infos.